charts/fluid/fluid/templates/role/dataset/rbac.yaml

```diff
@@ -61,6 +64,16 @@ rules:
       - statefulsets/status
     verbs:
       - '*'
+  - apiGroups:
```

Comment:
Is necessary to enlarge the RBAC authorization of dataset controller?

Reply:
done